File: java/Shirt.java
class Shirt extends Clothing{
    private String color;
    private String sleeveType;

    // Konstruktor default untuk membuat objek Kemeja dengan nilai awal
    Shirt()
    {
        
    }

    // Konstruktor untuk membuat objek Kemeja baru dengan detail yang diberikan
    public Shirt( String color, String sleveType) {
        this.color = color;
        this.sleeveType = sleveType;
    }

    // Getter dan setter untuk atribut color
    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    // Getter dan setter untuk atribut sleeveType
    public String getSleeveType() {
        return sleeveType;
    }

    public void setSleeveType(String sleeveType) {
        this.sleeveType = sleeveType;
    }

}

File: CPP/Shirt.cpp
#include <bits/stdc++.h>
#include "Clothing.cpp"
using namespace std;

class Shirt : public Clothing
{
    private:
    string color;
    string sleeveType;

    public:
    Shirt()
    {
        this->color = "";
        this->sleeveType = "";
    }

    Shirt(string color, string sleeveType)
    {
        this->color = color;
        this->sleeveType = sleeveType;
    }

    string getColor()
    {
        return this->color;
    }

    void setColor(string color)
    {
        this->color = color;
    }

    string getSleeveType()
    {
        return this->sleeveType;
    }

    void setSleeveType(string sleeveType)
    {
        this->sleeveType = sleeveType;
    }

    ~Shirt()
    {
        
    }
};